toolkit/tools/internal/timestamp/timestamp_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Test cases for timestamp.go.
+
+package timestamp
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+)
+
+var (
+	info1 = New("tool 1")
+	// info2 = New("tool 2", false)
+)
+
+//TestMain found in configuration_test.go.
+
+func initLogger() {
+	logger.InitStderrLog()
+}
+
+func Test_WritetoFile_range_instant(t *testing.T) {
+	initLogger()
+	info1.Start()
+	info1.RecordToFile("test step", "test action", os.Stdout)
+}
+
+// func Test_WritetoFile_noRange_instant(t *testing.T) {
+// 	info2.Start()
+// 	time.Sleep(30 * time.Millisecond)
+// 	info2.RecordToFile("test step", "test action", os.Stdout)
+// }
+
+func Test_WritetoCSV_range(t *testing.T) {
+	initLogger()
+	InitCSV("build-time.csv")
+	Stamp.Start()
+	time.Sleep(10 * time.Millisecond)
+	Stamp.RecordToCSV("step 1", "action 1")
+	Stamp.Start()
+	time.Sleep(20 * time.Millisecond)
+	Stamp.RecordToCSV("step 2", "action 1")
+	Stamp.Start()
+	time.Sleep(10 * time.Millisecond)
+	Stamp.RecordToCSV("step 2", "action 2")
+	Stamp.Start()
+	time.Sleep(30 * time.Millisecond)
+	Stamp.RecordToCSV("step 3", "action 1")
+}
```

Comment:
Here and below I'm also not sure what are we testing and how are we verifying the results.

Reply:
Deleting this file for this PR, will include a unit test file in a new PR.